packages/shared/styles/global.ts

```diff
@@ -4,27 +4,16 @@ const { fontFamilySansSerif } = coreFonts();
 
 export const injectGlobalCss = () => {
   return injectGlobal`
-    body {
-      font-family: ${fontFamilySansSerif};
-    }
-
-    h1, h2, h3, h4, h5, h6 {
-      font-weight: normal;
-      margin: 0;
-    }
-
-    p {
-      margin: 0;
-    }
-
-    ul, ol {
-      list-style: none;
-      margin-left: 0;
-      padding-left: 0;
-    }
-
     .ReactVirtualized__Grid {
       outline: none;
     }
   `;
 };
+
+export const injectStorybookResetCss = () => {
+  return injectGlobal`
+    body {
+        font-family: ${fontFamilySansSerif};
```

Comment:
Components look really weird when they're in the browser's default serif font. I'm comfortable with some minimal styling on anything rendered in Storybook instead of exporting this style with the components.